test_scripts/API/VehicleData/TireStatusMismatch/common.lua

```diff
@@ -0,0 +1,108 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local runner = require('user_modules/script_runner')
+local utils = require("user_modules/utils")
+
+--[[ General configuration parameters ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local m = {}
+
+--[[ Common Proxy Functions ]]
+m.Title = runner.Title
+m.Step = runner.Step
+m.preconditions = actions.preconditions
+m.postconditions = actions.postconditions
+m.start = actions.start
+m.registerApp = actions.app.register
+m.registerAppWOPTU = actions.app.registerNoPTU
+m.activateApp = actions.app.activate
+m.getMobileSession = actions.getMobileSession
+m.getHMIConnection = actions.hmi.getConnection
+m.policyTableUpdate = actions.policyTableUpdate
+m.cloneTable = utils.cloneTable
+m.spairs = utils.spairs
+m.isTableEqual = utils.isTableEqual
+m.tableToString = utils.tableToString
+
+--[[ Common Variables ]]
+m.tirePressureParams = {
+  "pressureTelltale", "leftFront", "rightFront", "leftRear", "rightRear", "innerLeftRear", "innerRightRear"
```

Comment:
@VjKlepikov  - So are we sending all the params for default/non-default tests? Is there a test where we skip each of these params or more in signal OVD/GVD response?

Reply:
It seems the test 1 and 2 above only test non-mandatory params for `SingleTireStatus`. Can you please add the tests which omit params like `leftRear`, `leftFront` etc.

Reply:
@atiwari9 
Please notice delivery PR for review is `https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2560`
Current PR was created just for Luxoft team internal review and was merged in main feature branch before delivery.

Regarding your comments:
Tests in current PR cover only cases for old applications (<=7.1)
In these scenarios omitted sub-parameters of `TireStatus` struct would be defaulted.

Existing VD tests cover all the cases for a new applications (>=8.0)
These tests would automatically take into account the changes in Mobile API ([rpc_spec](https://github.com/smartdevicelink/rpc_spec/compare/72632f946941d63a57ee5e99896e3eae3627f7dd...44f0a419c6a43815bc095134712eab4fef352ad7#diff-42ba46e1d01a876881e11902ad4a762c5040d0e7b089b17e7bfaaad9b10cd932) posted in `https://github.com/smartdevicelink/sdl_core/pull/3583`) and would generate appropriate scenarios.
In these scenarios omitted sub-parameters would be omitted as well in appropriate message for mobile application (response or notification)

As for new scripts in this PR:
- `001` and `002` - cover the following scenarios:
     - One sub-parameter is missing (iterating through all of them)
     - All sub-parameters are missing
     - All sub-parameters are present
 - `003` - covers scenario with 2 applications of different versions (`7.1` and `8.0`). In notification for old app omitted parameters would be defaulted, however for a new app those would be omitted

Reply:
Ok makes sense, i'd try to run the generic vehicle data test set to ensure that this is covered. Can you also please give that a run when possible.